README.md

```diff
@@ -32,38 +32,45 @@ There are 2 analyses using Pseudomonas dataset in the `Pseudomonas` directory an
 | [Human_experiment_lvl_sim](Human/Human_experiment_lvl_sim.ipynb) | Human experiment-level gene expression simulation|
 
 
-## Computational Environment
+## How to run notebooks from simulate-expression-compendia
 
-All processing and analysis scripts were performed using the conda environment specified in `environment.yml`.
-To build and activate this environment run:
+In order to run this simulation on your own gene expression data the following steps should be performed:
 
+First you need to set up your local repository: 
+1. Clone the `simulate-expression-compendia` repository
+2. Set up conda environment by running the following command in the terminal:
 ```bash
 # conda version 4.6.12
 conda env create -f environment.yml
 
 conda activate simulate_expression_compendia
-```
 
-Install local modules into this environment:
-```
-pip install .
+pip install -e .
 ```
+3. Navigate to either the `Pseudomonas` or `Human` directories and run the notebooks.
 
-Note: R scripts were run using R 3.6.3 
 
-## How to run this simulation using your own data
+## How to run using your own data
 
 In order to run this simulation on your own gene expression data the following steps should be performed:
 
-First you need to set up your local repository: 
+First you need to set up your local repository and environment: 
 1. Clone the `simulate-expression-compendia` repository
-2. Set up conda environment using the command above
+2. Set up conda environment by running the following command in the terminal:
```

Comment:
It may help to link to the miniconda install link here for people who don't know what `conda` is

Reply:
Good point. I will add this. Definitely modeled these new instructions after @miltondp 's nice example in phenoplier!